### ISSUE TITLE
[Mobile Payments] [Location API] If store has no line2 address, fetching location fails

### ISSUE THREAD

**Issue by allendav** [Automattic/woocommerce-payments#3012]
This is a must fix before the mobile apps can deploy Stripe Terminal SDK 2

My store has no line2 entry for its address. When I attempt to fetch a location from `payments/terminal/locations/store`, I get a failure, that I presume is from Stripe:

```
message	String?	"Error: You passed an empty string for 'address[line2]'. We assume empty values are an attempt to unset a parameter; however 'address[line2]' cannot be unset. You should remove 'address[line2]' from your request or supply a non-empty value."	some
```

Internal discussion: pdfdoF-as-p2

cc @ctarda @astralbodies

**Comment by kalessil** [comment]
@allendav, should be fixed with https://github.com/Automattic/woocommerce-payments/pull/2990 (merged yesterday).

**Comment by allendav** [comment]
I tested it pre-release pdfdoF-as#comment-462 and will test again with 3.1

**Comment by allendav** [comment]
This works really well. No line2 error. Tested with the released 3.1. Thank you!

<img width="830" alt="no-line-2" src="https://user-images.githubusercontent.com/1595739/136473902-5002efc0-b034-44b4-925c-4c4253ffa44e.png">
